src/Converter.php

```diff
@@ -29,6 +29,25 @@ public function getAsText(): string
         return $output;
     }
 
+    public function getSubstringHtml(string $numberOfCharacter, bool $escape = false): string
+    {
+        $output = '';
+        $checkOutput = '';
+        
+        foreach ($this->rows as $row) {
+            $checkOutput .= $row.PHP_EOL;
```

Comment:
Why do you use `PHP_EOL`? It's a useless character to count. What do you think about?

Reply:
Yeah it's typo from a previous version of the code, i'll remove in next commit :)